### PR TITLE
Service call adapter

### PR DESCRIPTION
**Pull request by franklinv77**

### CODE REVIEW

**Comment by ashish-formant** [comment]
why is this PR still open ?
@AaronGBoswell Can you review this PR? .Thanks

**Comment by fvolcic** [comment]
@AaronGBoswell we can delete this PR and I'll recreate it. Its not quite finished it needs to be adjusted for the new JSON format

File: examples/ros-service-call-adapter/adapter/services.py [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+        self._is_shutdown = True
+        # self._service_thread.join()
+
+    def _shutdown(self):
+        """return True if self._is_shutdown is set."""
+
+        return self._is_shutdown
+
+    def _run(self):
```
Comment (AaronGBoswell) [inline examples/ros-service-call-adapter/adapter/services.py:55]: Does anything call this?

File: examples/ros-service-call-adapter/adapter/setup.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        if "export CATKIN_WS" in line:
+            catkin_ws = line.split("=")
+            if len(catkin_ws) != 2:
+                exit(0)
+
+            catkin_ws = catkin_ws[1].lstrip().rstrip().replace("\n", "")
+
+if exists(f"{catkin_ws}/devel/setup.bash"):
+    print(f"{catkin_ws}/devel/setup.bash")
+    # print("")
```
Comment (AaronGBoswell) [inline examples/ros-service-call-adapter/adapter/setup.py:23]: If we just source /var/lib/formant/.bashrc in the setup, doesn't that do this?

**Pull request merged**